src/Widgets/Common/GoogleMapsWidget.php

```diff
@@ -86,31 +88,43 @@ protected function getTemplateData($widgetSettings, $isPreview)
 
         $curl = curl_init();
 
-        curl_setopt_array($curl, array(
-            CURLOPT_RETURNTRANSFER => 1,
-            CURLOPT_URL => $url
-        ));
+        curl_setopt_array(
+            $curl,
+            array(
+                CURLOPT_RETURNTRANSFER => 1,
+                CURLOPT_URL => $url
+            )
+        );
 
         $result_json = curl_exec($curl);
         $result = json_decode($result_json, true);
 
         curl_close($curl);
 
-        $lat = isset($result["results"][0]["geometry"]["location"]["lat"]) ? $result["results"][0]["geometry"]["location"]["lat"] : "";
-        $lng = isset($result["results"][0]["geometry"]["location"]["lng"]) ? $result["results"][0]["geometry"]["location"]["lng"] : "";
+        $lat = $result['results'][0]['geometry']['location']['lat'] ?? '';
+        $lng = $result['results'][0]['geometry']['location']['lng'] ?? '';
 
-        if ($lat && $lng)
-        {
+        if ($lat && $lng) {
             return [
-                "location" => [
-                    "lat" => $lat,
-                    "lng" => $lng
+                'location' => [
+                    'lat' => $lat,
+                    'lng' => $lng
                 ]
             ];
         }
 
+        if (isset($result['error_message'])) {
+            $this->getLogger(__CLASS__)->error(
+                'Google Maps API error',
+                [
+                    'status' => $result['status'],
+                    'error message' => $result['error_message']
```

Comment:
```suggestion
                    'error' => $result['error_message']
```

Du kannst den Key auch "message" nennen, Hauptsache keine Leerzeichen im Key.